P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -11,43 +11,47 @@ $Id$
+      of pages in a manuscript, or the layout of a printed page. This is not meant to be the only way
+      of transcribing primary sources in TEI, or even a preferred way; which approach is more appropriate
+      will depend on the specific needs of your project.</p>
+ 
+   <p>Although this chapter discusses manuscript materials more frequently than other forms of written text,
+      most of the recommendations presented are equally applicable to facsimiles of a wide variety of media,
+      including printed matter, monumental inscriptions, and art. Each medium has its own vocabulary of agents.
+      In the following examples, terms such as <soCalled>scribe</soCalled>, <soCalled>author</soCalled>, <soCalled>editor</soCalled>,
+      <soCalled>annotator</soCalled> or <soCalled>corrector</soCalled> may be re-interpreted in terms 
+      more appropriate to the medium being transcribed. In printed material, for example, the <soCalled>compositor</soCalled> plays a role analogous to the <soCalled>scribe</soCalled>, while in an authorial manuscript, the author and the scribe are the same person.</p>
+   
+   <p>This module may be used in conjunction with other modules. These recommendations are not intended to meet every transcriptional circumstance ever likely to be faced by any scholar. They are intended rather as a base to enable encoding of the most common phenomena found in the course of scholarly transcription of primary source materials. These guidelines particularly do not address the encoding of physical description of textual witnesses: the materials of the carrier, the medium of the inscribing implement, the organisation of the carrier materials themselves (as quiring, collation, etc.), authorial instructions or scribal markup, etc., except insofar as these are involved in the broader question of manuscript description, as addressed by the <ident type="module">msdescription</ident> module described in chapter <ptr target="#MS"/>. </p>
+   
+   <div xml:id="PHOV">
+      <p>This chapter begins by describing elements for handling digitally-encoded images of primary source materials for the purpose of creating digital facsimiles using the <gi>facsimile</gi> element (<ptr target="#PHFAX"/>).</p>
+      <p>The next section (<ptr target="#PH-transcr"/>) describes two ways of combining facsimile images with a transcription; either by referencing a parallel transcription in <gi>text</gi>, or by providing an <soCalled>embedded</soCalled> transcription that continues to prioritizing the encoding of a resource’s spatial features via the <gi>sourceDoc</gi> element and a number of transcriptional elements. </p>
+      <p>Section <ptr target="#PHST"/> expands on the <soCalled>embedded</soCalled> transcription concept by documenting elements that support scholars in recording information about specific features of the text on its physical carrier, such as <ptr target="#PHCH"/> and <ptr target="#PHPH"/></p>
```

Comment:
I have problems with "expands on the embedded transcription" because most of the referenced elements can be used in both, embedded and parallel transcription. Also, some of the presented examples are only possible in a parallel transcription, e.g.'But `<sic>`One`</sic>` must have lived'

Reply:
Thanks for catching this! You're absolutely correct.